--- kalsamix-1.0.0beta2/src/kalsamix.desktop.orig	2007-05-03 21:10:22.000000000 +0200
+++ kalsamix-1.0.0beta2/src/kalsamix.desktop	2008-05-06 16:07:11.000000000 +0200
@@ -1,7 +1,11 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=kalsamix
+Name[pl]=kalsamix
 Exec=kalsamix
 Icon=kalsamix
 Type=Application
 Comment=An ALSA mixer for KDE
+Comment[pl]=Mikser ALSA dla KDE
+GenericName[pl]=Mikser ALSA dla KDE
+Categories=Qt;KDE;Audio;Mixer;
